indigo_api/importers/pdfs.py

```diff
@@ -9,7 +9,10 @@ def pdf_count_pages(fname):
     """ Counts the number of pages in a PDF.
     """
     result = subprocess.run(["pdfinfo", fname], stdout=subprocess.PIPE, check=True)
-    m = re.search(r'^Pages:\s*(\d+)', result.stdout.decode('utf-8'), re.MULTILINE)
+    try:
+        m = re.search(r'^Pages:\s*(\d+)', result.stdout.decode('utf-8'), re.MULTILINE)
```

Comment:
Simpler to tell decode not to error, see https://docs.python.org/3.10/library/stdtypes.html#bytes.decode

eg. `stdout.decode('utf-8', 'ignore')`

Reply:
or `'replace'` is better because at least we can see that there's badness.